src/main/java/org/jboss/xavier/integrations/route/VMWorkloadInventoryRoutes.java

```diff
@@ -0,0 +1,36 @@
+package org.jboss.xavier.integrations.route;
+
+import org.apache.camel.builder.RouteBuilder;
+import org.jboss.xavier.integrations.migrationanalytics.business.VMWorkloadInventoryCalculator;
+
+import javax.inject.Named;
+
+@Named
+public class VMWorkloadInventoryRoutes extends RouteBuilder {
+   
+    @Override
+    public void configure() {
+        from("direct:calculate-vmworkloadinventory")
+                .id("calculate-vmworkloadinventory")
+                .doTry()
+                    .transform().method(VMWorkloadInventoryCalculator.class, "calculate(${body}, ${header.MA_metadata})")
+                    .split(body())
+                    .to("jms:queue:vm-workload-inventory")
+                .endDoTry()
+                .doCatch(Exception.class)
+                    .to("log:error?showCaughtException=true&showStackTrace=true")
+                    .setBody(simple("Exception on parsing Cloudforms file"))
+                .end();
+        
+        from ("jms:queue:vm-workload-inventory")
+                .id("extract-vmworkloadinventory")
+            .to("log:INFO?showBody=true&showHeaders=true")
+            .transform().method("decisionServerHelper", "generateCommands(${body}, \"GetWorkloadInventoryReports\", \"WorkloadInventoryKSession0\")")
+            .to("direct:decisionserver")
+            .transform().method("decisionServerHelper", "extractReports")
```

Comment:
[`extractReports`](https://github.com/project-xavier/xavier-integration/blob/master/src/main/java/org/jboss/xavier/integrations/DecisionServerHelper.java#L74) method extracts and returns `ReportDataModel` and it's not going to work with `WorkloadInventoryReportModel`.
For the ICS report the [`extractInitialSavingsEstimationReportModel`](https://github.com/project-xavier/xavier-integration/blob/master/src/main/java/org/jboss/xavier/integrations/DecisionServerHelper.java#L88) method has been created and something similar should be created in this case as well.

Reply:
+1 , naming confussed me...... ReportModel seems totally a super class for anything beign called ****ReportModel

Reply:
Added method